--- origsrc/writerperfect-0.9.5/configure.ac	2016-03-03 09:03:46.000000000 -0600
+++ src/writerperfect-0.9.5/configure.ac	2016-07-25 15:29:14.026844700 -0500
@@ -377,7 +377,7 @@ AM_CONDITIONAL([GSF_HAS_COMPRESSION_LEVE
 # libeot
 # ======
 wpft_CHECK_LIB([EOT], [libeot], [libeot], [Allow EOT->truetype conversion of embedded fonts])
-AS_IF([test "x$have_libgsf" = "xyes"], [EOT_CPPFLAGS=-DENABLE_EOT])
+AS_IF([test "x$have_libeot" = "xyes"], [EOT_CPPFLAGS=-DENABLE_EOT])
 AC_SUBST([EOT_CPPFLAGS])
 
 # ================
